src/main/java/com/icthh/xm/ms/entity/config/ApplicationProperties.java

```diff
@@ -136,4 +139,16 @@ public static class Jpa {
         private Integer findOneByIdForUpdateTimeout = 10000;
     }
 
+    @Getter
+    @Setter
+    public static class Elastic {
```

Comment:
remove elastic config, this config migrated to ee-conf @osiievska23

Reply:
No problem

it requires the next change, then:
`@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)`-> `@ConfigurationProperties(prefix = "application")`

The reason is ElasticProperties from wrapper package has mapping prefix `application.elastic`

Reply:
done